Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,118 @@
+
+h1 {
+	text-align: center;
+	line-height: 0.5;
+	font-size: 20px;
+	font-family: arial;
+	padding-bottom: 8px;
+}
+
+h2 {
+	text-align: center;
+	font-size: 35px;
+	line-height: 0.05;
+	font-family: arial;
+	padding-bottom: 30px;
+}
+
+h3 {
+	text-align: left;
```

Comment:
Unless the element is nested inside of another element of which you have declared a `text-align` of something other than `left`, `text-align: left;` is the default, so you don't really need to declare it